go.mod

```diff
@@ -53,7 +52,6 @@ require (
 )
 
 replace (
-	github.com/gardener/gardener-extensions => github.com/gardener/gardener-extensions v0.0.0-20191028142629-438a3dcf5eca
```

Comment:
Still need this replace, as gardener-resource-manager indirectly requires the extensions.

Reply:
Also please revendor :)

Reply:
Pinned g/grm dependency to `v0.9.0` which does not vendor g/g anymore. Fixed now. Nice catch.